### DISCUSSION TITLE
FlyCI: a faster and more affordable alternative to GitHub's MacOS M1 (and M2) runners!

### DISCUSSION THREAD

**Discussion by kgantchev** [community/community#86568] [Actions] [answered]
### Select Topic Area

Show & Tell

### Body

Hi, I'm the CEO and Co-Founder of [FlyCI](https://flyci.net). We offer a faster and more affordable alternative to GitHub's MacOS runners! We also provide M2 runners, which GitHub currently doesn't provide.

| Processor | vCPU | RAM (GB) | Storage | Label | Price on FlyCI | Price on GitHub |
|-----------|-------|-----------|----------|-------------------------------|----------------|-----------------|
| M1        | 4    | 7        | 28 GB | flyci-macos-large-latest-m1   | $0.06          | $0.08       |
| M1        | 8    | 14       | 28 GB | flyci-macos-xlarge-latest-m1  | $0.12          | $0.16           |
| M2        | 4    | 7        | 28 GB | flyci-macos-large-latest-m2   | $0.08          |      -       |
| M2        | 8    | 14       | 28 GB | flyci-macos-xlarge-latest-m2  | $0.16          |      -       |

Simply [install the FlyCI GitHub app](https://github.com/apps/flyci-prod) and replace the `runs-on` resource identifier from the current GitHub identifier you use with the FlyCI identifier. 

For example: 

```diff
-    runs-on: macos-latest
+    runs-on: flyci-macos-large-latest-m2
```

The utilization of FlyCI M1 runners (4vCPU) is free for public repositories, provided adherence to our <a href="https://www.flyci.net/#fair-usage-policy" target="_blank">Fair Usage Policy</a>.

For private repositories, we have a one-time offer of 200 free min for M1 runners (4vCPU).

**Comment by Akash1134** [comment, marked as answer]
Hi @kgantchev , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
